src/account/SecuritySubmenu.tsx

```diff
@@ -0,0 +1,514 @@
+import React, { useEffect, useRef } from 'react'
```

Comment:
This file is essentially just copy-pasting subsets of the functionality from the existing `Settings.tsx` file, basically no "new" stuff happening here.